lua/dylib.lua

```diff
@@ -11,6 +11,12 @@ local ffi = require("ffi")
 -- Structs
 ffi.cdef[[
 
+static const int LIST_IPV4_LPM_TABLE = 1;
+static const int LIST_IPV6_LPM_TABLE = 2;
+static const int LIST_LPM_TABLES = 3;
```

Comment:
Defining `list_fib4_entries()` and `list_fib6_entries()` instead of `list_fib_entries()` avoids defining constants `LIST_IPV4_LPM_TABLE`, `LIST_IPV6_LPM_TABLE`, and `LIST_LPM_TABLES` in C and in Lua.